admin-dev/themes/new-theme/js/components/module-card.js

```diff
@@ -347,11 +347,11 @@ export default class ModuleCard {
           self.confirmPrestaTrust(result[moduleTechName]);
         }
 
-        $.growl.error({message: result[moduleTechName].msg});
+        $.growl.error({message: result[moduleTechName].msg, fixed: true});
         return;
       }
 
-      $.growl.notice({message: result[moduleTechName].msg});
+      $.growl.notice({message: result[moduleTechName].msg, duration: 15000});
```

Comment:
```suggestion
      $.growl.notice({message: result[moduleTechName].msg, duration: 6000});
```

Should be way enough :p

Reply:
You mean I was too greedy?!
I'm OK !